src/chess/position_test.cc

```diff
@@ -0,0 +1,47 @@
+/*
+  This file is part of Leela Chess Zero.
+  Copyright (C) 2018 The LCZero Authors
+
+  Leela Chess is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Leela Chess is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with Leela Chess.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <gtest/gtest.h>
+
+#include <iostream>
+#include "src/chess/position.h"
+
+namespace lczero {
+
+TEST(PositionHistory, ComputeLastMoveRepetitions) {
```

Comment:
This test should probably be called ComputeLastMoveRepetitionsForPawnPushWithoutEnpassant (or similar)

And ideally we should have a ComputeLastMoveRepetitionsforPawnPushWithLegalEnpassant to cover both sides.